README.md

```diff
@@ -1,5 +1,5 @@
 
-# Grid Styled
+# Grid-Styled
```

Comment:
Thanks for the PR, but I'd prefer not to make a stylistic change to the name like this

Reply:
Agreed. I made a change to my PR.